### DISCUSSION TITLE
Obtaining Severity of Illness scores

### DISCUSSION THREAD

**Discussion by karen-gishyan** [MIT-LCP/mimic-code#1582] [MIMIC-III]
Dear MIMIC-III Community Members,

I would like to know how to obtain scores describing the severity of the illness for the patients. Also, I would like to know if such measures are available at different stages/time steps of the recovery process, which is needed for the research project I am currently conducting. In the paper, it is mentioned "The repository has been seeded with code to calculate commonly utilized variables in critical care research, including severity of illness scores,", so if you could provide guidance on how these scores can be obtained I would be very grateful.  Thanks in advance.

**Comment by yongfanbeta** [comment]
[severity of illness scores in mimiciii](https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iii/concepts/severityscores)

if you want to calculate those socres at different time, you can change the range of measurements charttime in the code.

**Comment by karen-gishyan** [comment]
Thanks. I assume the range of measurements is this line `DATETIME_ADD(ie.intime, INTERVAL '1' DAY)`, right?

**Reply by yongfanbeta** [reply]
Yes
